### PR TITLE
[4.2] revert 4.3er changes out of 4.2-dev

### PR DESCRIPTION
**Pull request by zero-24**
@tecpromotion @heelc29  please double check this PR. With this PR I would like to revert the 4.3er changes merged into 4.2 once we merge this PR and do an upmerge to 4.3 we need to re apply said changes to 4.3-dev

### CODE REVIEW

File: script.php [hunk 1]
```diff
@@ -24,7 +24,7 @@ class Pkg_deDEInstallerScript extends InstallerScript
 	 */
 	public function __construct()
 	{
-		// Define the minimum versions to be supported.
+		// Define the minumum versions to be supported.
```
Comment (tecpromotion) [inline script.php]:
```suggestion
		// Define the minimum versions to be supported.
```

**Review by tecpromotion — DISMISSED** [review]
Many Thanks Tobi. 

I would maybe even just leave the alphabetical sorting as it is.
So I would leave the changes as they are now.

**Comment by zero-24** [comment]
> I would maybe even just leave the alphabetical sorting as it is.
> So I would leave the changes as they are now.

Can you please clarify? I have "just" manually reverted the changes merged into 4.2 that should be only merged into 4.3.

Do you mean to close this PR and keep the changes merged and released and not revert them back?

**Review by tecpromotion — DISMISSED** [review]

**Comment by tecpromotion** [comment]
> > I would maybe even just leave the alphabetical sorting as it is.
> > So I would leave the changes as they are now.
> 
> Can you please clarify? I have "just" manually reverted the changes merged into 4.2 that should be only merged into 4.3.
> 
> Do you mean to close this PR and keep the changes merged and released and not revert them back?

Just my com_media commit.

**Comment by tecpromotion** [comment]
@heelc29 please also check this pr.

**Comment by zero-24** [comment]
So this @tecpromotion : https://github.com/joomlagerman/joomla/pull/2691/commits/727e74cf7fa42f571def7d5d371b8fd6cf3a0a60 ?

**Comment by tecpromotion** [comment]
> So this @tecpromotion : [727e74c](https://github.com/joomlagerman/joomla/commit/727e74cf7fa42f571def7d5d371b8fd6cf3a0a60) ?

see [c110d00](https://github.com/joomlagerman/joomla/pull/2691/commits/c110d00dec9d533d5d931c16f9ab3c3808bc3063)

**Review by tecpromotion — DISMISSED** [review]

File: administrator/language/de-DE/com_tags.ini [hunk 2]
```diff
@@ -42,8 +42,7 @@ COM_TAGS_EMPTYSTATE_BUTTON_ADD="Erstes Schlagwort erstellen"
 COM_TAGS_EMPTYSTATE_CONTENT="Schlagworte in Joomla! bieten eine flexible Möglichkeit, um Inhalte zu organisieren. Das gleiche Schlagwort kann auf viele verschiedene Einträge in verschiedenen Inhaltstypen angewendet werden."
 COM_TAGS_EMPTYSTATE_TITLE="Es wurden noch keine Schlagwörter erstellt."
 COM_TAGS_DELETE_NOT_ALLOWED="Das Löschen des Schlagworts „%s“ ist nicht erlaubt."
-COM_TAGS_ERROR_UNIQUE_ALIAS="Ein anderes Schlagwort hat denselben Alias."
-COM_TAGS_ERROR_UNIQUE_ALIAS_TRASHED="Ein anderes Schlagwort hat denselben Alias. Dieses Schlagwort befindet sich im Papierkorb."
+COM_TAGS_ERROR_UNIQUE_ALIAS="Ein anderes Schlagwort hat denselben Alias (ggf. ist es im Papierkorb)."
```
Comment (heelc29) [inline administrator/language/de-DE/com_tags.ini]:
```suggestion
COM_TAGS_ERROR_UNIQUE_ALIAS="Ein anderes Schlagwort hat denselben Alias."
COM_TAGS_ERROR_UNIQUE_ALIAS_TRASHED="Ein anderes Schlagwort hat denselben Alias. Dieses Schlagwort befindet sich im Papierkorb."
```

Included in #2647 merged in 4.2

**Review by heelc29 — APPROVED** [review]

**Comment by zero-24** [comment]
Ok will go and merge this up and send a PR to re apply this changes

**Comment by zero-24** [comment]
PR done here: https://github.com/joomlagerman/joomla/pull/2694#issue-1501978174

**Pull request merged**